### ISSUE TITLE
Dit can not read user.email and user.name from gitconfig

### ISSUE THREAD

**Issue by MariusVanDerWijden** [ditcraft/cli#13]
Hey guys, 
I am currently trying dit and I came across the following problem.

```console
$ dit init
err  > You don't have a user.email and/or user.name set in your git config. ditCLI won't work if this is not fixed!
err  > Please run 'dit config user.name <GITHUB_USERNAME_HERE>' and 'dit config user.email <GITHUB_EMAIL_HERE>'
err  > git is not configured correctly
$ dit config user.name MariusVanDerWijden
$ dit config user.email "mail"
$ dit init
err  > You don't have a user.email and/or user.name set in your git config. ditCLI won't work if this is not fixed!
err  > Please run 'dit config user.name <GITHUB_USERNAME_HERE>' and 'dit config user.email <GITHUB_EMAIL_HERE>'
$ git config -l
user.name=Marius van der Wijden
user.email=xxx
core.editor=nano
merge.tool=meld
core.repositoryformatversion=0
core.filemode=true
core.bare=false
core.logallrefupdates=true
remote.origin.url=https://github.com/ditcraft/cli
remote.origin.fetch=+refs/heads/*:refs/remotes/origin/*
branch.master.remote=origin
branch.master.merge=refs/heads/master
remote.marius.url=git@github.com:MariusVanDerWijden/cli
remote.marius.fetch=+refs/heads/*:refs/remotes/marius/*
user.name=MariusVanDerWijden
user.email=xxx

```
Edit: I've edit out my email, but it is a valid e-mail address

**Comment by marvinkruse** [comment]
Hey Marius,
thanks for letting us know. That should not happen - I'll look into this today and get back to you later!

**Comment by marvinkruse** [comment]
Hey Marius,

sorry for the late response. As we were a bit busy with other changes that included the handling of the git commands, but we now came to the conclusion that the function that checks whether a users git config is correct or not is not needed anymore. As we are only forwarding the git-related actions to the git client itself it's not really our job to verify the config of the user.

In our latest commit and release we fixed this issue, so you shouldn't run into it anymore. 🤞

Thanks again for your bug report, it really helped us as we ourselves didn't really run into this problem.
